src/ai/constants.rs

```diff
@@ -39,7 +41,8 @@ pub const AI_HELP_GPT3_5_FULL_DOC_NEW_PROMPT: AIHelpConfig = AIHelpConfig {
 
 pub const AI_HELP_GPT4_FULL_DOC_NEW_PROMPT: AIHelpConfig = AIHelpConfig {
     name: "20240125-gpt4-full_doc-new_prompt",
-    model: "gpt-4-0125-preview",
+    model: "gpt-4o-2024-05-13",
+    token_model: "gpt-4-0125-preview",
```

Comment:
We need to specify a separate `token_model`, because `tiktoken-rs` doesn't seem to support `gpt-4o` yet.